rfcs/0043-rfcsc-rotation.md

```diff
@@ -0,0 +1,102 @@
+---
+feature: rfcsc-rotation
+start-date: 2019-04-24
+author: Robin Gloster <mail@glob.in>, Simon Lackerbauer <mail@ciil.io>
+related-issues: 36
+---
+
+# Summary
+[summary]: #summary
+
+Each RFC Steering Committee (RFCSC) unanimously elects the succeeding one at
+their first meeting in December from an open list of nominees.
+
+# Motivation
+[motivation]: #motivation
+
+With the implementation of [RFC
+36](https://github.com/NixOS/rfcs/blob/master/rfcs/0036-rfc-process-team-amendment.md)
+the RFC Steering Committee has been established. Among others, future work for
+that RFC included a definition of how members to the Committee are to be chosen
+or removed. This RFC provides mechanisms through which membership in the
+Steering Committee can be established or ended, so as not to make membership a
+mandate for life, allowing a healthy rotation of members, and offering several
+procedures to always keep the Steering Committee capable of making decisions
+such that the RFC process can keep running smoothly at all times.
+
+# Detailed design
+[design]: #detailed-design
+
+The RFC Steering Committee shall always have five members. If membership drops
+below five members (for example by resignation of a member as detailed below), a
+new member shall directly be elected after a nomination period of at least two
+weeks (see below for nomination and selection process). If the number of
+members of the RFCSC drops below 4 people, it cannot proceed with shepherd team
+selections until new members have been selected.
+
+## Ending membership
+A member can end their membership in the Steering
+Committee by either of the following four mechanisms, ordered from most
+frequent to least:
+
+1. At the end of an election period.
+2. Resignation
+3. Unanimous vote by all other members after having missed two or more regular
+   meetings without giving an appropriate excuse.
```

Comment:
Doesn't have to be on the same meeting, vote would be cast per member.

Reply:
Do you mean to add that
> If the number of members of the RFCSC drops below 4 people

it also shouldn't be able to apply (3) or (4)?

If it was about members being _present_, I thought it's clear that "all other" means all other _on the committee_ (not just present ATM).

Reply:
I think the natural interpretation here is «all the other current members», I think reaching 3 instead of 2 is unfortunate and reaching 4 is outright bad whatever you do, and I think in both cases it is less bad to thave the possibility of removal before selecting replacement.